REVIEWER.md

```diff
@@ -1,20 +1,21 @@
 As a NeuroLibre reviewer, you are responsible for the technical quality of the resources available for our community. Neurolibre welcome submissions along two tracks of neuroscience-related material: (1) tutorials, (2) paper companions. Prior to review, an editor establishes that the submission qualifies in principles, and an administrator has made the resource available for the neurolibre binder, so you can review the material directly on our portal (the link is at the top of the `README.md` file). Now your role is to ensure the submitted materials take full advantage of the notebook format, prior to final publication. Specific criteria for review are listed below.
 
 # Technical review Criteria
-Examples of high quality tutorials can be found in the scikit-learn documentation, for example this one on [cross-validation](https://scikit-learn.org/stable/modules/cross_validation.html). Examples of high quality article companions can be found as collab links in the article [building blocks of interpretability](https://distill.pub/2018/building-blocks/). Specific areas for review include:
```

Comment:
I think we should point the one specific notebook, and not the whole article because reviewers review a notebook. (if we point to the whole article, then we should talk about the jupyter book).

Reply:
In that particular example it's hard to read the notebooks outside of the context of the paper. It's also about the way authors use notebooks to enhance a publication, and reviewers should have a look at that as well. In other words, track 2 is not only about having great notebooks, but also notebooks that complement well a publication. Or maybe that's too much work, and we want reviewers to look at notebooks as stand alone ojects?

Reply:
I think the reviewers should look at each notebook on his own (he is reviewing "the code" inside a notebook), the whole article (jupyter book in our case) is just a way to enhance visibility,
any thought on this @emdupre ?